src/modules/auth/utils/login-error-parser.util.ts

```diff
@@ -27,5 +27,5 @@ function getStatusErrorMessage(statusCode: number) {
 }
 
 function getNetworkErrorMessage() {
-  return 'Our servers are offline at the moment. Please try again later';
+  return 'Failed to connect to backend. Server might be offline or configured incorrectly';
```

Comment:
The old message was more user-friendly, actually. Not everyone know what backend is. =)